runtime/src/lib.rs

```diff
@@ -8,39 +8,35 @@
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
-use babe::AuthorityId as BabeId;
-use client::{
-    block_builder::api::{self as block_builder_api, CheckInherentsResult, InherentData},
-    decl_runtime_apis, impl_runtime_apis, runtime_api as client_api,
-};
-use grandpa::fg_primitives;
-use grandpa::{AuthorityId as GrandpaId, AuthorityWeight as GrandpaWeight};
-use primitives::{crypto::key_types, OpaqueMetadata};
+use primitives::OpaqueMetadata;
```

Comment:
Is this our implementation of the node-template file updated to the latest released?

Reply:
Not sure I understand your question, but yes, I did follow the changes in the node-template file. We are using a bunch more than the default template though.